src/integrationtests/java/com/ericsson/ei/integrationtests/flow/FlowStepsIT.java

```diff
@@ -88,10 +88,12 @@
     private JenkinsXmlData jenkinsXmlData;
     private SubscriptionObject subscriptionObject;
     private JSONObject jobStatusData;
+    public String AggEvent;
```

Comment:
Please give a better name instead of AggEvent

Reply:
follow java naming standards for variables